### ISSUE TITLE
add support for extension less JavaScript / CSS files (JS files "in disguise")

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#700]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
As part of testing out integration with Material Web Components in https://github.com/ProjectEvergreen/greenwood-getting-started/pull/42, it was observed that some tools [are not including an extension](https://github.com/material-components/material-web/issues/2632) which causes a couple issues
- CSS files that are supposed to be treated as JS files will not return correctly
- JS files that are actually there will not return correctly

Basically, MWC will reference [a CSS file as such in JavaScript](https://unpkg.com/browse/@material/mwc-button@0.25.2/mwc-button.js)
```js
.
.
import { styles } from './styles.css';
```

But the actual file exists as [_styles.css.js_](https://unpkg.com/browse/@material/mwc-button@0.25.2/styles.css.js).

## Details
As this is non standard behavior, the CSS support should be put into **plugin-import-css** and for JavaScript, I think it is fine (for now) to include in **plugin-node-modules** since bare specifiers are the norm there.

**Comment by thescientist13** [comment]
Or at least document something since this kind of error for a missing _.js_ extension is definitely not helpful / insightful
![greenwood-faq-no-file-extension-console](https://user-images.githubusercontent.com/895923/132359375-38ce4629-647a-4655-bd01-afa6006bb8f7.png)
<img width="1611" alt="greenwood-faq-no-file-extension" src="https://user-images.githubusercontent.com/895923/132359388-4f27a4db-8be0-489a-a5b7-9c7bf8f59e0b.png">
